node/blockchain/migrations/0002_tablelock.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.2.9 on 2021-12-07 13:51
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('blockchain', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='TableLock',
+            fields=[
+                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
```

Comment:
let's use MongoDB built-in `_id`, and preferably keep it autogenerated by Mongo

Reply:
It use MongoDB build-in _id by default

`
test_node> db.blockchain_lock.find()
[
  {
    _id: ObjectId("61b3768411b0fc044db46d3b"),
    name: 'blockchain_block'
  }
]
`

Reply:
Done.

Reply:
Django works strange with objects who has unique field with "_id" name

`_id = models.PositiveBigIntegerField(primary_key=True, auto_created=True)`

I can create new model. But when I tried to delete such model I got next error message:
`AssertionError: Lock object can't be deleted because its _id attribute is set to None.`

I think we need to keep Lock model without custom primary key field. Without custom primary key Django has a "id" and MongoDB has "_id":

`test_node> db.blockchain_lock.find()
[
  {
    _id: ObjectId("61b3768411b0fc044db46d3b"),
    name: 'blockchain_block'
  }
]
`